src/UI/state/home/actions/getHomepageStats.ts

```diff
@@ -0,0 +1,23 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { fetchGraphQlData } from '../../../api/api';
+import { getHomepageAnalytics } from '../../../api/queries';
+import { updateStats, serverValidation } from '../homeSlice';
+
+const date = new Date();
+
+export const getHomepageStats = createAsyncThunk(
+  'home/getHomepageStats',
+  async (_, thunkAPI) => {
+    try {
+      const response = await fetchGraphQlData(
+        getHomepageAnalytics(0, date.getTime(), 'hour', 'Europe%2FLondon')
+      );
+      const data = response.data.getHomepageAnalytics;
+      thunkAPI.dispatch(serverValidation(true));
+      thunkAPI.dispatch(updateStats(data));
+    } catch (e: any) {
+      console.log(e.message);
```

Comment:
Console log should be removed...

Reply:
Don't we want to inform technical people if something has failed?

Reply:
Use the logger (in utils) - this does admittedly just console log it, but we'll swap it for something 'proper' later